src/Eagle.ts

```diff
@@ -2104,14 +2110,23 @@ export class Eagle {
     addNodeToLogicalGraph = (node : Node) : void => {
         //console.log("addNodeToLogicalGraph()", node.getName(), node.getCategory(), node.getInputPorts().length, node.getOutputPorts().length, node.getFields().length);
 
+        
         // get new position for node
-        let pos = this.getNewNodePosition();
+        if (Eagle.nodeDropLocation.x == 0 && Eagle.nodeDropLocation.y == 0){
+            Eagle.pos = this.getNewNodePosition();
+        }else if (Eagle.nodeDropLocation){
```

Comment:
Would this ever be false? I guess that Eagle.nodeDropLocation will always be defined.